### DISCUSSION TITLE
Error 500 on Restore

### DISCUSSION THREAD

**Discussion by MynameisStevetoo** [snipe/snipe-it#15281] [Q&A]
So I managed to get everything up and running, looking good. I imported all my users and then the assets, forgot to check that it was to update current records and it duplicated everything, DOH!

No problem, I have backups..... until I tried to restore... 

Now I get a 500 error. 

All that comes up on the logs is:

```
2024-08-13 16:24:30	Access	xx.xx.xx.xx.xx	302	GET /login HTTP/1.0
2024-08-13 16:24:30	Error	xx.xx.xx.xx.xx	500	GET /setup HTTP/1.0
```

Running Ubuntu 24.04 LTS with all the latest versions of Snip-IT and dependencies. 

Am I best to wide the DB and start fresh? A bit of a pest, but meh, it happens.

**Comment by snipe** [comment]
Those look like your access logs - we need to see what’s in the `storage/logs/laravel.log`

**Comment by MynameisStevetoo** [comment]
Sorry you're right! 

Plot twist, I just refreshed, the page....and pre-fight kicked in again....how odd....sorry that's the second time today I've wasted your time! ha ha ha

**Reply by snipe** [reply]
I was at dinner anyway, so no time was wasted on my end :D
